portability-core/src/test/java/org/dataportabilityproject/serviceProviders/flickr/FlickrPhotoServiceTest.java

```diff
@@ -0,0 +1,126 @@
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class FlickrPhotoServiceTest {
+
+  static final String PHOTO_TITLE = "Title";
+  static final String FETCHABLE_URL = "fetchable_url";
+  static final String DESCRIPTION = "Description";
+  static final String MEDIA_TYPE = "jpeg";
+
+  static final String ALBUM_ID = "Album ID";
+  static final String ALBUM_NAME = "Album name";
+  static final String ALBUM_DESCRIPTION = "Album description";
+
+  static final PhotoModel PHOTO_MODEL = new PhotoModel(PHOTO_TITLE, FETCHABLE_URL, DESCRIPTION,
+      MEDIA_TYPE, ALBUM_ID);
+  static final PhotoAlbum PHOTO_ALBUM = new PhotoAlbum(ALBUM_ID, ALBUM_NAME, ALBUM_DESCRIPTION);
+
+  Flickr flickr = Mockito.mock(Flickr.class);
+  PhotosetsInterface photosetsInterface = Mockito.mock(PhotosetsInterface.class);
+  PhotosInterface photosInterface = Mockito.mock(PhotosInterface.class);
+  Uploader uploader = Mockito.mock(Uploader.class);
+  JobDataCache jobDataCache = Mockito.mock(JobDataCache.class);
+  User user = Mockito.mock(User.class);
+  Auth auth = new Auth(Permission.WRITE, user);
+  FlickrPhotoService photoService = new FlickrPhotoService(flickr, photosetsInterface,
+      photosInterface, uploader, jobDataCache, auth);
+
+  @Test
+  public void getPage() {
+    // No pagination information, so we must be on page 1
+    assertThat(FlickrPhotoService.getPage(Optional.empty())).isEqualTo(1);
+
+    // Correctly get current page
+    int page = 6;
+    FlickrPaginationInformation paginationInformation = new FlickrPaginationInformation(page);
+    assertThat(FlickrPhotoService.getPage(Optional.of(paginationInformation))).isEqualTo(page);
+  }
+
+  @Test
+  public void getMimeType() {
+    assertThat(FlickrPhotoService.toMimeType("jpeg")).isEqualTo("image/jpeg");
+    assertThrows(IllegalArgumentException.class, () -> FlickrPhotoService.toMimeType("png"));
+  }
+
+  @Test
+  public void toCommonPhoto() {
+    Photo photo = new Photo();
+    photo.setTitle(PHOTO_TITLE);
+    Size size = new Size();
+    size.setSource(FETCHABLE_URL);
+    size.setLabel(Size.ORIGINAL);
+    List<Size> sizeList = Arrays.asList(size);
+    photo.setSizes(sizeList);
+    photo.setDescription(DESCRIPTION);
+    photo.setOriginalFormat(MEDIA_TYPE);
+
+    PhotoModel photoModel = FlickrPhotoService.toCommonPhoto(photo, ALBUM_ID);
+    assertThat(photoModel.getAlbumId()).isEqualTo(ALBUM_ID);
+    assertThat(photoModel.getFetchableUrl()).isEqualTo(FETCHABLE_URL);
+    assertThat(photoModel.getTitle()).isEqualTo(PHOTO_TITLE);
+    assertThat(photoModel.getDescription()).isEqualTo(DESCRIPTION);
+    assertThat(photoModel.getMediaType()).isEqualTo("image/jpeg");
+  }
+
+  @Test
+  public void exportAlbumReturnsNextPage() throws IOException, FlickrException {
+    ExportInformation emptyExportInfo =
+        new ExportInformation(Optional.empty(), Optional.empty());
+
+    // Set up auth
+    Mockito.when(user.getId()).thenReturn("id");
```

Comment:
In tests, we may want to static import the Mockito.when and other static methods to be consistent with the junit assert* static imports.

Reply:
Done.